pkgs/applications/networking/cluster/kubectl-explore/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kubectl-explore";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "keisku";
+    repo = "kubectl-explore";
+    rev = "v${version}";
+    hash = "sha256-4WxvVsA05Mta7AcrGe26B+Up+x/gwdlCnb/PN9Ehu18=";
+  };
+
+  vendorSha256 = "sha256-z/bPfY9UVqOnrA9jNUtM7jg53/URAMAJQAqH9D5KVPQ=";
```

Comment:
```suggestion
  vendorHash = "sha256-z/bPfY9UVqOnrA9jNUtM7jg53/URAMAJQAqH9D5KVPQ=";
```

Reply:
Hello, I've just pushed code with the change

I'm not sure what's the benefit of `vendorHash` though. I only found the following sentence (in [the docs](https://nixos.wiki/wiki/Nix_Hash)): 

_An alternative – and supposedly preferred – way of specifying hashes are so-called "SRI hashes"._ 

Could you provide more context why it's the preferred way?

Reply:
Sure!

Basically hashes can be generated from different hashing algorithms. By using `vendorHash` it means that the hash you're using must be a SRI hash, which is prefixed by the hashing algorithm in use. This change is optional, but since it's your first PR, let's use the best practice from the beginning 🤗

Reply:
Cool, thanks for explaining